packages/in_app_purchase/in_app_purchase_android/lib/src/billing_client_wrappers/sku_details_wrapper.dart

```diff
@@ -68,8 +72,15 @@ class SkuDetailsWrapper {
   final String introductoryPrice;
 
   /// [introductoryPrice] in micro-units 990000
```

Comment:
Can we add a new comment saying something similar to the android doc:
/// Returns 0 if the SKU is not a subscription or doesn't have an introductory period.

Reply:
Done, good catch. I think that is a valuable addition.